templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,42 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List
+  </h1>
+  <p><h5><a class="link" href="{% url 'taxi:manufacturer-create' %}">Click here to add a new manufacturer</a></h5></p>
```

Comment:
Code Style: According to the checklist, you should use 2 whitespaces for indentation in your `.html` files. Please, adjust your code to follow this rule.